### DISCUSSION TITLE
AMazon Product URL tracking

### DISCUSSION THREAD

**Discussion by sudip696** [towfiqi/serpbear#86] [Ideas]
Is there any way to track keywords for specific URLs
for exa:
[www.amazon.com/Patchouli-Premium-Therapeutic-Grade-Essential/dp/B00QJCVD04](https://www.amazon.com/Patchouli-Premium-Therapeutic-Grade-Essential/dp/B00QJCVD04)

Keywords:
Patchouli Oil
Patchouli Essential Oil

I want to track the keywords ranking for the above URL.

**Comment by sudip696** [comment]
@towfiqi

**Comment by towfiqi** [comment]
Sorry, the way the app has been built, its not possible to track a single url.
